src/amo/components/Errors/AuthExpired/index.js

```diff
@@ -0,0 +1,69 @@
+/* @flow */
+/* global window */
+import * as React from 'react';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+
+import ErrorComponent from 'amo/components/Errors/ErrorComponent';
+import translate from 'amo/i18n/translate';
+import Button from 'amo/components/Button';
+import type { I18nType } from 'amo/types/i18n';
+import { logOutUser as logOutUserAction } from 'amo/reducers/users';
```

Comment:
`import type { I18nType } from 'amo/types/i18n';` is the only type import?

Reply:
> Also, is there a reason it's necessary to rename this from the default of `logOutUser` to `logOutUserAction`?

I don't think so but we did that in a couple of other files (e.g. amo/components/App) so I thought it was best practise 🤷

Reply:
> `import type { I18nType } from 'amo/types/i18n';` is the only type import?

Also `import type { DispatchFunc } from 'amo/types/redux';`

Reply:
> > Also, is there a reason it's necessary to rename this from the default of `logOutUser` to `logOutUserAction`?
> 
> I don't think so but we did that in a couple of other files (e.g. amo/components/App) so I thought it was best practise 🤷

It might be that we've done that with this action creator in other places, but for the most part we do not do it, so I think it's unnecessary.